caddyhttp/markdown/metadata/metadata.go

```diff
@@ -68,7 +68,7 @@ func (m *Metadata) load(parsedMap map[string]interface{}) {
 		switch v := val.(type) {
 		case bool:
 			m.Flags[key] = v
-		case string:
+		default:
```

Comment:
@mholt: I think they have to: you can access variables through ```{{ .Doc.Name }}``` and flags through ```{{ .DocFlags.Name }}```. I could of course change it so flags and variables will be accessible through ```{{ .Doc.Name }}``` only but it will break for existing clients.